q-and-a.md

```diff
@@ -0,0 +1,48 @@
+
+## Wat is het verdienmodel?
+
+Nuts is een samenwerkingsverband van softwareleveranciers en heeft daarom ook geen verdienmodel. Door samen te werken aan een gedeelde infrastructuur pakt elke leverancier een deel van de rekening op. Dit laat zich terugbetalen in een betere concurrentiepositie. Immers een betere informatievoorziening is voor de zorgprofessional een belangrijk punt waarop zij hun keuze voor leverancier baseren.
+
+## Hoe gaat het straks werken voor gebruikers?
+
+Zowel de patiënt als de zorgmedewerker zal niet direct met Nuts werken. De patiënt krijgt een Persoonlijke gezondheidsomgeving (PGO) zoals dat nu door [MedMij](https://medmij.nl) wordt uitgewerkt. Ook de medewerker blijft gewoon zelf een software leverancier kiezen voor de procesondersteuning. Op de achtergrond zullen gegevens tussen deze systemen worden uitgewisseld met behulp van Nuts. Het voordeel hiervan is dat patiënten toestemmingen één keer op uniforme wijze kunnen vastleggen in bijvoorbeeld hun PGO. Zowel de patient als het netwerk van zorgverleners kunnen meedoen in de dialoog. Dat is het voordeel van een universele infrastructuur.
+
+## Waarom is er niet al zoiets als Nuts?
+
+Deels omdat Nuts een kind van deze tijd is. We zien in de laatste jaren een aantal trends:
+* Het digitaal samenwerken in de zorg wordt steeds belangrijker.
+* Er is een steeds grotere behoeften aan systemen die privacy van de burgers kunnen waarborgen.
+* De technologie om decentraal en veilig informatie uit te wisselen wordt steeds volwassener.
+De bovenstaande punten vragen om een nieuwe aanpak en maken dat wij als leveranciers nu dit initiatief hebben opgestart. Iets wat een aantal jaren geleden nog niet mogelijk was.
+
+En deels misschien ook omdat het business model vrij lastig is. Het is een probleem waar iedereen last van heeft, maar zodra je er geld voor gaat vragen versnippert de markt en bereik je dus niet je doel. Dus moeten we dit pre-concurrentieel met elkaar oplossen, en marktwerking heeft daarbij eigenlijk in de weg gezeten.
+
+Dat gezegd hebbende: Nuts bestaat natuurlijk niet in een vacuum. Er zijn wel al heel erg veel mensen in Nederland bezig om stukjes van dit probleem op te lossen, en daar maken wij natuurlijk dankbaar gebruik van. Wij pakken alleen de stukjes op die volgens ons onderbelicht zijn gebleven, en waarvoor je elkaar echt nodig hebt.
```

Comment:
Er zijn al wel heel erg veel mensen in Nederland bezig om... --> Veel mensen zetten zich al in om

"stukjes" (verkleinend taalgebruik) vervangen door onderdelen of onderwerpen.